docker-compose.yml

```diff
@@ -24,5 +27,32 @@ services:
       - .parking-service.env
     volumes:
     - images:/cowbot-images
-    #ports:
-    #  - '3000:3000'
+
+  skill-search:
+    build:
+      context: ./skill-search
+    restart: unless-stopped
```

Comment:
Add an `image:` block here to tag the image as `fearlesstech/cowbot-skill-search:latest`, similar to line 22 above.